polkadot-parachains/Cargo.toml

```diff
@@ -95,3 +98,9 @@ runtime-benchmarks = [
 	'statemine-runtime/runtime-benchmarks',
 	'westmint-runtime/runtime-benchmarks',
 ]
+try-runtime = [
+	'polkadot-service/try-runtime',
```

Comment:
Do we really need support of this in Polkadot be enabled?

Reply:
copy pasta'd from `runtime-benchmarks`. Indeed not needed, fixed.